src/svgdom/util/style_stack_cache.cpp

```diff
@@ -0,0 +1,116 @@
+#include "style_stack_cache.hpp"
```

Comment:
By the way, do you use this `style_stack_cache` class somewhere in your programs? I'm thinking about moving it from `svgdom` to `svgren` in the future, possibly. In case it is too specific thing which is not needed for common use. Just asking, no need to do anything about it so far.

Reply:
I think it's important to keep `style_stack_cache` because it's not only used for rendering; if one would like to find bounding boxes for all elements to manipulate them, for example, the style-stacks would have to be used to determine the true position and bounds.

Reply:
Ok, I'll keep it in `svgdom`.